src/test/java/com/epam/tc/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,157 @@
+package com.epam.tc.hw2.ex1;
+
+import static java.util.Map.entry;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.hw2.BaseTest;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class FirstExerciseTest extends BaseTest {
+
+    private final Map<String, String> imageCssLocatorToTextAfterImage = Map.ofEntries(
+        entry(".icons-benefit.icon-practise",
+            "To include good practices\nand ideas from successful\nEPAM project"),
+        entry(".icons-benefit.icon-custom",
+            "To be flexible and\ncustomizable"),
+        entry(".icons-benefit.icon-multi",
+            "To be multiplatform"),
+        entry(".icons-benefit.icon-base",
+            "Already have good base\n(about 20 internal and"
+            + "\nsome external projects),\nwish to get more…")
+        );
+
+    private final List<String> expectedHeaderButtonsText = Stream
+        .of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS")
+        .sorted()
+        .collect(Collectors.toList());
+
+    private final String frameButtonValue = "Frame Button";
+    private final List<String> expectedLeftSectionItemsText = Stream
+        .of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs")
+        .sorted()
+        .collect(Collectors.toList());
+
+
+
+
+
+    @Test
+    public void firstExerciseTest() {
+        gotoSite();
+        checkPageTitle(expectedBrowserTitle);
+        login(user);
+        checkUserIsLoggined(user);
+        checkAllHeadersItemsDisplayed(expectedHeaderButtonsText);
+        checkImagesDisplayedAndHaveTextBelow(imageCssLocatorToTextAfterImage);
+        checkFrameWithButtonExists(frameButtonValue);
+        checkLeftSectionHasItemsWithProperText(expectedLeftSectionItemsText);
+    }
+
+
+
+    // 5. Assert that there are 4 items on the header section
+    // are displayed and they have proper texts;
+    public void checkAllHeadersItemsDisplayed(List<String> expectedHeaderButtonsText) {
+        List<WebElement> headerNavigationButtons = driver.findElements(
+            By.cssSelector(".uui-navigation.nav.navbar-nav > *")
```

Comment:
Weak locator. Can be modified to more stable.

Reply:
Modified to `nav > ul.uui-navigation.nav.navbar-nav.m-l8 > li`